.editorconfig

```diff
@@ -4,6 +4,8 @@ root = true
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
```

Comment:
Isn't golang tab-based?
I thought go is very specific in required style, but maybe I'm mistaken.

Reply:
its actually whatever gofmt says.
I have removed it for now.